### DISCUSSION TITLE
Where is the MH-60S Knighthawk???

### DISCUSSION THREAD

**Discussion by Triton08** [Project-Hatchet/public_h-60#373] [General]
I just loaded up into Eden Editor to check out the new functions with the 0.6 update from Sept 29th and under H-60 Pack in NATO i didnt see the 2 pylon MH-60S Knighthawk version for the US Navy..... what happened to it??? I want it back. that was my units go to H-60.  :( :(

**Comment by PDTFawks** [comment]
It was hidden from the editor/Zeus. You can still spawn it using the `createVehicle` command or making a config patch to un-hide it.
**However** it **will** have issues and is **not** recommended to use it. The Navy skins are still in the mod as well so you can reskin a normal UH-60 since they are basically the same and the Navy versions had nothing special to them anyway.
